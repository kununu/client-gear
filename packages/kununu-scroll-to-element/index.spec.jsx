import React from 'react';
import {render} from 'enzyme'; // eslint-disable-line import/no-extraneous-dependencies

import ScrollToElement from './index';


const Test = () => (<span>test</span>);

test('Renders ScrollToElement without crashing', () => {
  const component = render(
    <ScrollToElement
      className="class"
      duration={150}
      key={1}
      mobileOnly={false}
      tagName="article"
    >
      <Test />
    </ScrollToElement>,
  );

  expect(component).toMatchSnapshot();
});
