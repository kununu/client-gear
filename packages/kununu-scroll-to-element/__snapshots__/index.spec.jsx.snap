// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders ScrollToElement without crashing 1`] = `
<article
  class="class"
>
  <span>
    test
  </span>
</article>
`;
